groups/prometheus/main.tf

```diff
@@ -0,0 +1,76 @@
+provider "aws" {
+  region = var.region
+}
+
+terraform {
+  backend "s3" {}
+}
+
+module "security-groups" {
+  source           = "./module-security-groups"
+  region           = var.region
+  vpc_id           = local.vpc_id
+  ssh_cidrs        = concat(local.internal_cidrs,local.vpn_cidrs)
+  prometheus_cidrs = concat(local.mgmt_private_subnet_cidrs)
+  tag_environment  = var.tag_environment
+  tag_service      = var.tag_service
+}
+
+module "ec2-instances" {
+  source                 = "./module-ec2-instances"
+  instance_count         = var.instance_count
+  instance_type          = var.instance_type
+  ami_version_pattern    = var.ami_version_pattern
+  application_subnets    = local.mgmt_private_subnet_ids
+  vpc_security_group_ids = module.security-groups.vpc_security_group_ids
+  zone_name              = var.zone_name
+  ssh_keyname            = var.ssh_keyname
+  private_key_path       = var.private_key_path
+  tag_environment        = var.tag_environment
+  tag_service            = var.tag_service
+  web_fqdn               = var.web_fqdn
+  metrics_port           = var.metrics_port
+}
+
+module "route53-records" {
+  source                   = "./module-route53-records"
+  instance_count           = var.instance_count
+  zone_id                  = var.zone_id
+  zone_name                = var.zone_name
+  ec2_instance_private_ips = module.ec2-instances.ec2_instance_private_ip[*]
+  tag_environment          = var.tag_environment
+  tag_service              = var.tag_service
+}
+
+# ------------------------------------------------------------------------------
+# Locals
+# ------------------------------------------------------------------------------
+
+locals {
+  vpc_id                              = data.terraform_remote_state.management_vpc_eu_west_1.outputs.management_vpc_id
+  mgmt_private_subnet_cidrs           = values(data.terraform_remote_state.management_vpc_eu_west_1.outputs.management_private_subnet_cidrs)
```

Comment:
The region we deploy resources to should really be tied to the active profile region. Given that the `mgmt_private_subnet_cidrs` variable here is statically tied to the `eu-west-1` region (and this is ultimately the region in which the EC2 instance is deployed regardless of the profile region, albeit using the `application_subnets` variable), if we were to add a profile and environment `vars` file for `eu-west-2` it would similarly be tied to `eu-west-1`.

It's worth considering reworking this so that the value for `mgmt_private_subnet_cidrs` is dynamic based on the profile region, as it may save future work when we migrate everything to `eu-west-2`.